nixos/tests/6tunnel.nix

```diff
@@ -0,0 +1,41 @@
+import ./make-test-python.nix ({ pkgs, lib, ... } : {
+  name = "6tunnel";
+  meta.maintainers = with pkgs.lib.maintainers; [ ymarkus ];
+
+  nodes = {
+
+    client = { ... }: {
+      networking.enableIPv6 = false;
```

Comment:
... this has literally no effect.

Reply:
I tried doing a test like this:
```
client (IPv4 only) -> webhost (IPv4 only) fails
client -> tunnel (both) -> webhost succeeds
```
but I couldn't make it work.

Reply:
Maybe @Mic92, @andersk, or @tfc know something about this?